app/src/main/java/com/github/se/polyfit/ui/components/scaffold/SimpleTopBar.kt

```diff
@@ -22,7 +23,7 @@ fun SimpleTopBar(title: String, navigateBack: () -> Unit) {
       title = {
         Text(
             title,
-            modifier = Modifier.testTag("${titleCase(title)} Title"),
+            modifier = Modifier.testTag("${titleCase(title)} Title").fillMaxWidth(),
```

Comment:
Why fill max width? If its to center the title, see https://github.com/swent-group10/polyfit/blob/b4137f6448ef50988c1a05e0bf290bd28a543b44/app/src/main/java/com/github/se/polyfit/ui/components/scaffold/CenteredTopBar.kt coming in #187 which I think gives a better solution, since we don't always want title in the center.

Reply:
Its because for some reason that when I didn't fill the max Width the title didn't display at all.

Reply:
That seems unexpected, but I don't think this is the right solution, as this component is already working in other screens